src/components/Web3Status/AccountStatus.tsx

```diff
@@ -52,20 +53,22 @@ const Web3StatusConnected = styled.button<{ pending?: boolean }>`
   text-transform: uppercase;
   cursor: pointer;
   outline: none;
+  display: flex;
+  align-items: center;
 `
 
 const Web3StatusNetwork = styled.button<{ pendingTransactions?: boolean; isConnected: boolean; clickable: boolean }>`
   display: flex;
   align-items: center;
-  height: 25px;
+  height: 28px;
   padding: 7px 8px;
   font-size: 12px;
   line-height: 15px;
   text-align: center;
   letter-spacing: 0.08em;
   text-transform: uppercase;
   color: #ffffff;
-  border-radius: 12px;
+  border-radius: 10px;
```

Comment:
It Fixes the border-radius. Sadly, the spacing between the container and child components is now 4px instead of 3px. This is due to the nested `div` tree that requires some heavy refactoring.